### PR TITLE
Issue429

### PR DESCRIPTION
**Pull request by pjohans**

### CODE REVIEW

File: ting.entities.inc [hunk 1]
```diff
@@ -316,6 +317,10 @@ class TingEntity extends DingEntity {
       drupal_alter('ting_online_url', $url, $this);
     }
 
+    if(strpos($url, '[useraccessinfomedia]') === 0){
+      drupal_alter('infomedia_url', $url, $this);
+    }
+
```
Comment (kasperg) [inline ting.entities.inc]: This change seems to be irrelevant to the issue. It is a mistake that it made it into this pull request?

File: ting.client.inc [hunk 2]
```diff
@@ -687,8 +688,10 @@ function ting_execute($request) {
     }
 
     return $response;
-  }
-  catch (TingClientException $e) {
+  } catch (TingClientException $e) {
```
Comment (kasperg) [inline ting.client.inc]: Catch statements should be on a new line according to the code standards.

**Comment by kasperg** [comment]
Review complete. There are a couple of issues here.

**Comment by JrgenGNielsen** [comment]
@kasperg : This fixes the issue, I hope: [https://github.com/ding2/ting/pull/39]
